sycl/CMakeLists.txt

```diff
@@ -184,6 +184,21 @@ install(DIRECTORY ${OPENCL_INCLUDE}/CL
   COMPONENT opencl-headers
 )
 
+if (NOT TARGET LLVMGenXIntrinsics)
+  message(STATUS "vc-intrinsics are missing. Will try to download them from github.com")
+  include(FetchContent)
```

Comment:
The line #1 in this CmakeLists.txt file says:
cmake_minimum_required(VERSION 3.2)
But the FetchContent becomes available only in Cmake-3.11

Is there any specific reason why you need FetchContent and move the download to 'configure' phase instead of using ExternalProject and have the download at 'build' phase?

Reply:
We have a long discussion in an email tread. The reason for this is a cyclic dependency LLVM->vc-instrinsics->LLVM (LowerESIMD). And we decided that I can use this mechanism,

Reply:
Please add those people to reviewers.
If you keep using this mechanism, you would need to a) update the first line in this file and b) update the pre-requisites here: https://intel.github.io/llvm-docs/GetStartedGuide.html#prerequisites

Reply:
OK, done. I set 3.14 version because FetchContent_MakeAvailable was introduced at this version.